### ISSUE TITLE
E1101 false positive

### ISSUE THREAD

**Issue by NickeZ** [PyCQA/pylint#2229]
### Steps to reproduce
1. Create file with content:
```
from pathlib import Path
Path('/test/a').parent.is_file()
```
2. run pylint on file.

### Current behavior
```
E:  3, 0: Instance of 'PurePath' has no 'is_file' member (no-member)
```

### Expected behavior
No warning, `Path.parent` returns a `Path` not a `PurePath`
### pylint --version output
```
$ pylint --version
Using config file /home/niklas/.pylintrc
pylint 1.9.2,
astroid 1.6.3
Python 3.6.5 (default, Apr  1 2018, 05:46:30)
[GCC 7.3.0]
```

**Comment by brycepg** [comment]
You can find this fix in the `pylint 2.0` prelease:

`pip install --pre --upgrade pylint`
